### PR TITLE
Add unit tests to Story Promo 

### LINKED ISSUE #541: Add unit tests to Story Promo
**Is your feature request related to a problem? Please describe.**
At the moment we don't have tests that check if the `StoryPromo` component is rendering the right and expected elements.

**Describe the solution you'd like**
Add tests in `@psammead-story-promo/src/index.test.jsx` to check if the `StoryPromo` renders the `h3`, `p`, `time`, and `img` as done already in the Simorgh [container](https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.test.jsx).

Also, we should check if the Image is rendered with the `src` and `alt` attributes.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #541

**Overall change:** Add assertion unit tests to `StoryPromo` to check if the `StoryPromo` renders `h3`, `p`, `time`, and `img`

**Code changes:**

- Add `@testing-library/react` as a dev dependancy
- Add assertion unit tests
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-story-promo/package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "@bbc/psammead-test-helpers": "^0.3.3",
     "@bbc/psammead-timestamp": "^0.3.2",
     "react": "^16.8.6",
-    "styled-components": "^4.1.3"
+    "styled-components": "^4.1.3",
+    "@testing-library/react": "^8.0.1"
```
Comment (dr3) [inline packages/components/psammead-story-promo/package.json]: Can you install this at the top level of psammead instead please, would be good for every package to have it :) (and as tests are run from top level its fine :) )

Just make sure to bump the top level package, lock and changelog

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by dr3 — APPROVED** [review]

**Comment by staylos92** [comment]
This does not need testing by a tester

**Pull request merged**